### ISSUE TITLE
"Repeated commands" not working 

### ISSUE THREAD

**Issue by jerclarke** [rxhanson/RectanglePro-Community#78]
Rectangle Pro version: 2.4.7
macOS: 12.1.3
Device: MacBook Pro (16-inch 2019)

Hi! New user who loves RP very much here!

Normally I use the **Repeated commands move to adjacent display on left and right** setting, which gives the delightful effect of cycling through the different "slots" at the given width.But it's not working, and when I change the setting none of the other options work either. 

Similarly my "Custom Shortcuts" that have "2nd Execution" etc. aren't working. 

In all cases, subsequent invocations of the command result in the window kind of flashing out of place, like it's trying to take the next position, but then immediately flashing back into the position of the first invocation. 

It's super weird!

IMO it started when the app updated a few days ago. 

I have tried restarting my computer and the app several times.

Thanks for any help you can offer. Not sure if it's just me or maybe a larger problem.

<img width="736" alt="Screen Shot 2022-07-08 at 10 38 46 AM" src="https://user-images.githubusercontent.com/139995/178025098-3397d1e6-eb01-4f70-a4c8-19084d453c59.png">

**Comment by rxhanson** [comment]
Thanks for reporting this. I cannot reproduce it, but maybe there's something I'm missing here. 

First, let's take a look at the logs: Hold option, and in the menu bar menu select View Logging...

Reproduce the issue and paste the logs in here.

Next, maybe it would help if I loaded your config and gave it a try? Export your config and send it my way and I'll give it a go.

Is this occurring with all applications or just certain ones?

I don't think I made any changes in the most recent release that would have affected this behavior, but after seeing the logs and/or trying your config, I'm pretty sure I will be able to narrow down what's going on.

**Comment by jerclarke** [comment]
Thanks for the reply!

Here's the logs for clicking to Finder, then using **Last third** a few times:

```
2022-07-11T09:07:45-05:00: Obtained window from window number under cursor
2022-07-11T09:07:48-05:00: AX sizing proposed: (832.0, 1415.0), result: (625.0, 1415.0)
2022-07-11T09:07:48-05:00: AX position proposed: (1666.0, 25.0), result: (1666.0, 25.0)
2022-07-11T09:07:48-05:00: AX sizing proposed: (832.0, 1415.0), result: (831.0, 1415.0)
2022-07-11T09:07:48-05:00: lastThird | display: (0.0, 0.0, 2498.0, 1415.0), calculatedRect: (1666.0, 25.0, 832.0, 1415.0), resultRect: (1666.0, 25.0, 831.0, 1415.0), srcScreen: LG Ultra HD, destScreen: LG Ultra HD, resultScreen: LG Ultra HD
2022-07-11T09:07:49-05:00: AX sizing proposed: (832.0, 1415.0), result: (625.0, 1415.0)
2022-07-11T09:07:49-05:00: AX position proposed: (1666.0, 25.0), result: (1666.0, 25.0)
2022-07-11T09:07:49-05:00: AX sizing proposed: (832.0, 1415.0), result: (831.0, 1415.0)
2022-07-11T09:07:49-05:00: lastThird | display: (0.0, 0.0, 2498.0, 1415.0), calculatedRect: (1666.0, 25.0, 832.0, 1415.0), resultRect: (1666.0, 25.0, 831.0, 1415.0), srcScreen: LG Ultra HD, destScreen: LG Ultra HD, resultScreen: LG Ultra HD
2022-07-11T09:07:50-05:00: AX sizing proposed: (832.0, 1415.0), result: (625.0, 1415.0)
2022-07-11T09:07:50-05:00: AX position proposed: (1666.0, 25.0), result: (1666.0, 25.0)
2022-07-11T09:07:50-05:00: AX sizing proposed: (832.0, 1415.0), result: (831.0, 1415.0)
2022-07-11T09:07:50-05:00: lastThird | display: (0.0, 0.0, 2498.0, 1415.0), calculatedRect: (1666.0, 25.0, 832.0, 1415.0), resultRect: (1666.0, 25.0, 831.0, 1415.0), srcScreen: LG Ultra HD, destScreen: LG Ultra HD, resultScreen: LG Ultra HD
2022-07-11T09:07:54-05:00: Obtained window from window number under cursor
```

Here's an export of my config: https://gist.github.com/jerclarke/ddfddd722f9baa54cbcc9a189a74fd3d

It's happening with all applications as far as I can tell. Finder, Chrome, Terminal, for example.

Thanks for taking a look!

**Comment by rxhanson** [comment]
Sorry for the delay here. I loaded your config and repeated commands are working fine for me. 

Your logs show some strange behavior where windows are not being sized as requested by the app. I have typically seen this in two scenarios: 
1. Some apps don't resize exactly down to the pixel, like iTerm, which will resize by character sizes by default. Since you see this for all apps, this doesn't seem to be the issue here.
2. Another window manager is triggering resizing. Maybe make sure nothing else could be adjusting your windows somehow?

Beyond that, I'm honestly not sure what this could be but will let this sink in a little more and see if I can figure out what might cause this.

**Comment by jerclarke** [comment]
Ugh! I'm so sorry for wasting your time!

I still had Divvy installed, and somehow the box to start up at login had become re-ticked after I disabled it!

Removed the app completely to avoid this happening again. Thank you so much for taking the time to test my config and get back to me. 100% user error and I should have thought to re-check that there was no other app using the same shortcuts!!!

🙏🏻
